codalab/objects/metadata_spec.py

```diff
@@ -19,6 +19,15 @@ def bool_constructor(x=False):
     return x == 'True'
 
 
+def unicode_constructor(s=""):
```

Comment:
Makes me kind of sad that we have to do this.  Can we put a big TODO that gives context - like we basically blow away unicode, but we need to support unicode in the future.

Reply:
Yeah, on top of that I say we leave #502 open since this is more of a bandaid fix. We can definitely close #829 and #961 for now after this is merged.

Reply:
When we migrate to Python 3, good unicode handling will come for free. In addition, I suggest we adopt types, as it was really annoying how half of our strings are `str` and the other half is `unicode` but you have no idea when (even for the same function sometimes we get different typed inputs).